File: mot-selenium/src/main/java/uk/gov/dvsa/domain/model/mot/OdometerUnit.java
package uk.gov.dvsa.domain.model.mot;

public enum OdometerUnit {
    MILES("mi"),
    KILOMETRES("km");

    private final String value;

    OdometerUnit(String value) {
        this.value = value;
    }

    public String getValue(){
        return value;
    }
}

File: mot-common-web-module/src/DvsaCommon/Constants/OdometerUnit.php
<?php
/**
 * This file is part of the DVSA MOT API project.
 *
 * @link https://gitlab.motdev.org.uk/mot/mot
 */

namespace DvsaCommon\Constants;

/**
 * Class OdometerUnit
 *
 * Enums defined on mot_test_current.odometer_unit and certificate_replacement_draft.odometer_unit
 */
class OdometerUnit extends BaseEnumeration
{
    const MILES = 'mi';
    const KILOMETERS = 'km';
}
